<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Home" failures="1" tests="3" name="Test" time="0.468" errors="0" timestamp="2023-05-17T01:42:12 IST">
  <testcase classname="com.testLayer.Demo2" name="login" time="0.028"/>
  <testcase classname="com.testLayer.Demo2" name="verifyTitle" time="0.001"/>
  <testcase classname="com.testLayer.Demo2" name="verifyURL" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [nm] but found [nn]">
      <![CDATA[java.lang.AssertionError: expected [nm] but found [nn]
at com.testLayer.Demo2.verifyURL(Demo2.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyURL -->
</testsuite> <!-- Test -->
